content/en/docs/tutorials/security/seccomp.md

```diff
@@ -199,10 +199,10 @@ test workload execution before rolling the change out cluster-wide.
 More detailed information about a possible upgrade and downgrade strategy can be
 found in the [related Kubernetes Enhancement Proposal (KEP)](https://github.com/kubernetes/enhancements/tree/a70cc18/keps/sig-node/2413-seccomp-by-default#upgrade--downgrade-strategy).
```

Comment:
```suggestion
You can find more detailed information about a possible upgrade and downgrade strategy
in the related Kubernetes Enhancement Proposal (KEP):
[Enable seccomp by default](https://github.com/kubernetes/enhancements/tree/9a124fd29d1f9ddf2ff455c49a630e3181992c25/keps/sig-node/2413-seccomp-by-default#upgrade--downgrade-strategy).
```
Notice that I bumped to commit https://github.com/kubernetes/enhancements/commit/9a124fd29d1f9ddf2ff455c49a630e3181992c25